plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/building/N4JSBuildTypeTrackingBuilder.java

```diff
@@ -74,7 +82,25 @@ protected void doBuild(ToBeBuilt toBeBuilt, IProgressMonitor monitor, BuildType
 		try {
 			updateProjectReferencesIfNecessary();
 			N4JSBuildTypeTracker.setBuildType(getProject(), type);
-			super.doBuild(toBeBuilt, monitor, type);
+
+			SubMonitor sm = SubMonitor.convert(monitor, 100);
+			SubMonitor monitorMF = sm.newChild(1);
+			SubMonitor monitorPJ = sm.newChild(99);
+			super.doBuild(getManifestTBB(toBeBuilt), monitorMF, type);
+			List<String> errMsgsInManifest = getErrorMsgsInManifest();
+			if (errMsgsInManifest.isEmpty() || !isAbortBuildOnMfErrors()) {
+				// Only build the project if the manifest file is error free OR if setting is disabled
+				super.doBuild(toBeBuilt, monitorPJ, type);
+			} else {
+				String completeLogMsg = "\n";
+				completeLogMsg += "Errors in Manifest. Project is not built.\n";
+				completeLogMsg += "Manifest errors:\n";
+				for (String errMsg : errMsgsInManifest) {
+					completeLogMsg += errMsg + "\n";
+				}
+				LOGGER.warn(completeLogMsg);
```

Comment:
not sure if this should be logged as a warning.
From the perspective of IDE usage, having a project with errors in the manifest is an ordinary use case (e.g. happens all the time while editing/refactoring things)